addons/test_website/tests/test_controller_args.py

```diff
@@ -29,3 +31,18 @@ def test_crawl_args(self):
         req = self.url_open('/ignore_args/kw?a=valueA&b=valueB')
         self.assertEqual(req.status_code, 200)
         self.assertEqual(req.json(), {'a': 'valueA', 'kw': {'b': 'valueB'}})
+
+    def test_route_type(self):
+        with MockRequest(self.env):
```

Comment:
Do you need to use MockRequest here as you're not using the request object?

Reply:
no, indeed :)